client/orm/cmd.go

```diff
@@ -76,6 +77,7 @@ func RunCommand() {
 
 // sync database struct command interface.
 type commandSyncDb struct {
+	ctx       context.Context
```

Comment:
I think we don't need this `ctx` field. We could pass `context.Backgroud` if we need context.

From my understanding, we can do nothing with context when we run command fron temminal.

Reply:
this can be removable